tools/sof_perf_analyzer.py

```diff
@@ -274,8 +271,8 @@ def analyze_perf_info():
     perf_stats.columns = ['CPU_AVG(MIN)', 'CPU_AVG(AVG)', 'CPU_AVG(MAX)',
                           'CPU_PEAK(MIN)', 'CPU_PEAK(AVG)', 'CPU_PEAK(MAX)']
     perf_stats['PEAK(MAX)/AVG(AVG)'] = perf_stats['CPU_PEAK(MAX)'] / perf_stats['CPU_AVG(AVG)']
-    comp_peak_avg_cycles = perf_info.groupby('COMP_ID')['CPU_PEAK'].mean()
-    perf_stats['MODULE_CPC'] = comp_peak_avg_cycles * CPC_MARGIN
+    comp_peak_max_cycles = perf_info.groupby('COMP_ID')['CPU_PEAK'].max()
+    perf_stats['MODULE_CPC'] = comp_peak_max_cycles
```

Comment:
`perf_stats['MODULE_CPC'] = perf_info.groupby('COMP_ID')['CPU_PEAK'].max()` single line is good.

Let's make this commit a standalone PR, because
- it has nothing to do with widget id issue
- it can be merged quickly